src/less/dropdown/base/dropdown.less

```diff
@@ -12,7 +12,8 @@ span.menu,
 span.fake-menu,
 span.listbox,
 span.combobox {
-    display: inline-block;
+    display: inline-flex;
```

Comment:
I'm curious at to what changing it to flex layout gains us? I didn't think the immediate child elements of those needed to be flex items?

Reply:
Let me see about reverting this change since I added the `fixed-height` modifiers.